src/Raven.Server/NotificationCenter/Handlers/DatabaseNotificationCenterHandler.cs

```diff
@@ -6,11 +6,81 @@
 using Raven.Server.Documents;
 using Raven.Server.NotificationCenter.Notifications;
 using Raven.Server.Routing;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
 
 namespace Raven.Server.NotificationCenter.Handlers
 {
     public class DatabaseNotificationCenterHandler : DatabaseRequestHandler
     {
+        [RavenAction("/databases/*/notifications", "GET", AuthorizationStatus.ValidUser, EndpointType.Read, SkipUsagesCount = true)]
+        public async Task GetNotifications()
+        {
+            var postponed = GetBoolValueQueryString("postponed", required: false) ?? true;
+            var type = GetStringQueryString("type", required: false);
+            var start = GetIntValueQueryString("pageStart", required: false) ?? 0;
+            var pageSize = GetIntValueQueryString("pageSize", required: false) ?? int.MaxValue;
+            
+            var shouldFilter = type != null;
+            if (shouldFilter && Enum.TryParse(typeof(NotificationType), type, out _) == false)
```

Comment:
`NotificationType` enum is quite big:

https://github.com/ravendb/ravendb/blob/3bd416cc5d472653dc0a45e0fa06eeb7f0fb6228/src/Raven.Server/NotificationCenter/Notifications/NotificationType.cs#L3-L13

And the only relevant values here are `AlertRaised` and `PerformanceHint`, right? (because only those are persisted). I wonder if we should use this enum. Maybe better to have support for `type=[alert|performance-hint]`? @ppekrol?

Reply:
I would prefer to go with enum, but you are correct, this one is not good. Can we introduce a new one? Maybe even with [Flags] attribute, so we can pick multiple ones?